worker/worker.go

```diff
@@ -186,17 +186,15 @@ func (worker *Worker) Ready() (err error) {
 		return ErrNoneFuncs
 	}
 	for _, a := range worker.agents {
-		if err = a.Connect(); err != nil {
-			return
-		}
+		go a.Connect()
 	}
 
 	// `once` protects registering worker functions multiple times.
-	worker.once.Do(func() {
-		for funcname, f := range worker.funcs {
-			worker.addFunc(funcname, f.timeout)
-		}
-	})
+	//	worker.once.Do(func() {
```

Comment:
Rather than commenting out, delete and rely on git to recover if we decide we want it again.